docs/first-order/optotagging.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>I may be mistaken but I believe setting the apy key to an empty string throws an error for this dandiset, is that right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>